### PR TITLE
TASK: Refactor TYPO3.Neos to use PSR-4

### PR DESCRIPTION
**Pull request by johannessteu**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -33,6 +34,9 @@
         "psr-4": {
             "Neos\\Diff\\": [
                 "Neos.Diff/Classes"
+            ],
+            "TYPO3\\Neos": [
```
Comment (Benjamin-K) [inline composer.json]: Shouldn't this be `"TYPO3\\Neos\\": [ ... ]`?

**Comment by Benjamin-K** [comment]
Does the Vendor name really change? i think this is just a change from PSR-0 to PSR-4, isn't it?

File: composer.json (hunk 1 above)
Comment (kitsunet) [inline composer.json]: Yes it should

**Comment by kitsunet** [comment]
Vendor namespace?
Neos.Diff was Neos vendor since the beginning...
And I don't see a vendor change

**Comment by Benjamin-K** [comment]
So the Label should be `!!! PSR change` or sth.

**Comment by kitsunet** [comment]
@Benjamin-K I don't get it? This is not a breaking change.

**Comment by Benjamin-K** [comment]
@kitsunet you are right, but the Label (currently `!!! Vendor name change`) is incorrect, so this should be changed.

**Comment by kitsunet** [comment]
oh the label, yes maybe, but it's related to that, so for us it's better for tracking those changes.

**Comment by johannessteu** [comment]
Yeah, related to the vendor name change as it is a prepration for it

**Comment by Benjamin-K** [comment]
Oh, i see. Nevermind then ;)

**Comment by robertlemke** [comment]
Closing this pull request because the PSR-4 migration will be done by @kitsunet as part of the vendor namespace change.

**Pull request closed without merging**